src/styles/modules/_m-headings.sass

```diff
@@ -2,47 +2,47 @@
   +element('h1')
     +rem-fallback('font-size', $font-size-xl)
     font-weight: 700
-    font-family: $font-helvetica
+    font-family: $font
     color: color(agray, dark)
-    +rem-fallback('line-height', $line-height-h2)
-    +rem-fallback('letter-spacing', $letter-spacing-xs)
+    +rem-fallback('line-height', $line-height-xl)
+    +rem-fallback('letter-spacing', $letter-spacing-default)
     +rem-fallback('margin-bottom', 20px)
 
     +breakpoint(tablet-wide)
       +rem-fallback('font-size', $font-size-xxl)
-      +rem-fallback('line-height', $line-height-h1)
+      +rem-fallback('line-height', $line-height-xxl)
 
-    &--museo
-      font-family: $font-museo
+    &--special
+      font-family: $font
```

Comment:
special should be `Bayu Prahara, Bradley Hand ITC, cursive` not the same as $font

Reply:
Good catches, Becky! I pushed a fix for this and your next two comments.